src/components/drawer/DrawerContent.tsx

```diff
@@ -15,7 +15,7 @@ const DrawerContent: FC<PropsWithChildren<Props>> = (props) => {
 
   return (
     <ChakraDrawerContent {...drawerContentProps}>
-      {withCloseButton && <DrawerCloseButton />}
+      {withCloseButton && <DrawerCloseButton style={{ fontSize: '14px' }} />}
```

Comment:
We usually try to leverage the chakra props and also our tokens. 14px would be `sm` but I think this is too small. Touch areas should be at least 16px and for the modal we also use 16px for the close button.

```suggestion
      {withCloseButton && <DrawerCloseButton fontSize="base" />}
```